Makefile

```diff
@@ -27,10 +27,21 @@ image:
 	docker build -f public.Dockerfile -t ${IMAGE}:${VERSION} .
 	docker tag ${IMAGE}:${VERSION} ${IMAGE}:latest
 
+.PHONY: image-version
+image-version:
+	docker build -f public.Dockerfile -t ${IMAGE}:${VERSION} .
+
+.PHONY: push-image-version
+push-image-version:
+	docker push ${IMAGE}:${VERSION} .
+
 .PHONY: push-image
 push-image:
-	docker push ${IMAGE}:${VERSION}
-	docker push ${IMAGE}:latest
+	docker push ${IMAGE}:latest .
+
+.PHONY: image-version
+image-version:
+	docker build -f public.Dockerfile -t ${IMAGE}:${VERSION} .
 
 .PHONY: oidc-image
```

Comment:
do we still need this image given we remove oidc in metadata? cc @verdan

Reply:
Yes we still need this. The only thing we removed from metadata was extra OIDC config class. We still need to set `flaskoidc` package to enable oidc in metadata. and this image will be doing that.